dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -107,6 +116,15 @@ def make_project_dataframe_from_query(self, context: QueryContext, project_confi
         table_handler = PivotedTableHandler(project_config, dataset_id=self.dataset_id)
         return table_handler.convert_columns_to_query_names(ld_df)
 
+    @staticmethod
+    def _add_null_values(ld_df, null_lk_df):
+        if not null_lk_df.rdd.isEmpty():
+            for col in set(ld_df.columns).difference(null_lk_df.columns):
+                null_lk_df = null_lk_df.withColumn(col, F.lit(None))
```

Comment:
I see why you need null timestamps now

Reply:
The dataset contributor is the one who sets the schema. We do check this in `check_null_value_in_unique_dimension_rows(dim_table, exclude_columns=None)`